scripts/onnx/test.sh

```diff
@@ -59,36 +60,29 @@ if [[ "${SHARD_NUMBER}" == "1" ]]; then
   # run them locally
   pytest "${args[@]}" "${args_parallel[@]}" \
     --ignore "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py" \
-    --ignore "$top_dir/test/onnx/test_custom_ops.py" \
     --ignore "$top_dir/test/onnx/test_models_onnxruntime.py" \
-    --ignore "$top_dir/test/onnx/test_utility_funs.py" \
-    --ignore "$top_dir/test/onnx/test_pytorch_onnx_caffe2.py" \
-    --ignore "$top_dir/test/onnx/test_pytorch_onnx_shape_inference.py" \
     --ignore "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime_cuda.py" \
-    --ignore "$top_dir/test/onnx/test_pytorch_onnx_caffe2_quantized.py" \
+    --ignore "$top_dir/test/onnx/test_custom_ops.py" \
+    --ignore "$top_dir/test/onnx/test_utility_funs.py" \
+    --ignore "$top_dir/test/onnx/test_models.py" \
     "${test_paths[@]}"
 
-  # Tests that cannot run in parallel.
+  # Heavy memory usage tests that cannot run in parallel.
   pytest "${args[@]}" \
     "$top_dir/test/onnx/test_models_onnxruntime.py" \
     "$top_dir/test/onnx/test_custom_ops.py" \
-    "$top_dir/test/onnx/test_utility_funs.py"
-
-  pytest "${args[@]}" "${args_parallel[@]}" \
-    "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset7" \
-    "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset8" \
-    "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset9" \
-    "$top_dir/test/onnx/test_pytorch_onnx_shape_inference.py" \
-    "$top_dir/test/onnx/test_pytorch_onnx_caffe2.py" \
-    "$top_dir/test/onnx/test_pytorch_onnx_caffe2_quantized.py"
+    "$top_dir/test/onnx/test_utility_funs.py" \
+    "$top_dir/test/onnx/test_models_onnxruntime.py" "-k" "not TestModelsONNXRuntime"
```

Comment:
This `-k` thing is a bit confusing. What's left in this file besides `TestModelsONNXRuntime`? Why not move it to another file?

Reply:
There are two other test classes running model export test with ort defined from `test_model.py`. `test_model.py` tests with caffe2. I'm opening #79802 to track this.